test/terra/backends/test_pulse_simulator.py

```diff
@@ -684,6 +686,8 @@ def test_frame_change(self):
         exp_prop_shift = self._analytic_prop_fc(np.pi / 4 - np.pi / 8)
         self.assertDictAlmostEqual(prop_shift, exp_prop_shift, delta=0.01)
 
+    '''
```

Comment:
So the proper way to do this, is by using the `@unittest.skipTest("PerisitentValue pulses are currently not supported")`

Reply:
Ah nice, thanks. I've made this change :).